src/cmd/rio/menu.c

```diff
@@ -80,6 +82,34 @@ Menu	egg =
 	version
 };
 
+void
+runterm() {
+	if(termprog != NULL){
+		execl(shell, shell, "-c", termprog, (char*)0);
```

Comment:
Isn't the "-c" option specific to some shells ? (I remember it vaguely from bash)

Reply:
-c is specific to all posix compliant shells...

http://pubs.opengroup.org/onlinepubs/009695399/utilities/sh.html

AFAIK, the shells below have this option:
sh, rc, bash, csh, dash, fish, zsh, nash

Reply:
I think that if you recite the sequence of shell names above very fast you get brain damage kkkk

Reply:
auheauheauheuaheuhauheuahe

Did not know that, thanks =)